vector/v.cluster/main.c

```diff
@@ -493,7 +493,7 @@ int main(int argc, char *argv[])
 
             mean = sum / n;
             sd = sqrt(sumsq / n - mean * mean);
-            eps = mean + 1.644854 * sd; /* 90% CI */
+            // eps = mean + 1.644854 * sd; /* 90% CI */
```

Comment:
There is another duplicate `eps` in line 290.

Somehow the code is unexpected, can @metzm enlighten us?

Reply:
Done removed another line as well